homeassistant/components/braviatv/__init__.py

```diff
@@ -44,3 +67,237 @@ async def async_unload_entry(hass, config_entry):
 async def update_listener(hass, config_entry):
     """Handle options update."""
     await hass.config_entries.async_reload(config_entry.entry_id)
+
+
+class BraviaTVClient(DataUpdateCoordinator[None]):
+    """Representation of a Bravia TV Client.
+
+    An instance is used per device to share the same power state between
+    several platforms.
+    """
+
+    def __init__(self, hass, host, mac, pin, ignored_sources):
+        """Initialize Bravia TV Client."""
+
+        self.hass = hass
```

Comment:
```suggestion
```

Reply:
Hmmm, it used :)

Reply:
It's set by the parent.
https://github.com/home-assistant/core/blob/17414439dfa27485ed486b91c618898eea7995b9/homeassistant/helpers/update_coordinator.py#L47